app/views/projects/workflow_executions/show.html.erb

```diff
@@ -94,68 +94,93 @@
   <% end %>
 <% end %>
 
-<%= render Pathogen::TabsPanel.new(id: "workflow-executions-tabs", label: t(:".tabs.label")) do |tabs| %>
+<%= render Pathogen::Tabs.new(
+  id: "workflow-executions-tabs",
+  label: t(:".tabs.label"),
+  default_index: @tab_index,
+  sync_url: true
+) do |tabs| %>
   <% tabs.with_tab(
     id: "summary-tab",
-    text: t(:"workflow_executions.show.tabs.summary"),
-    href:
-      namespace_project_workflow_execution_path(
-        @namespace.parent,
-        @namespace.project,
-        @workflow_execution,
-        tab: "summary",
-      ),
-    selected: @tab == "summary" || @tab == nil,
-    controls: "workflow-content",
+    label: t(:"workflow_executions.show.tabs.summary"),
+    selected: @tab == "summary" || @tab == nil
```

Comment:
Suggestion (blocking): no need for nil check

Reply:
Updated in ddfc086776b08530b3effaecbd6c55cb058601d4